osxcollector/osxcollector.py

```diff
@@ -912,20 +924,15 @@ def _log_sqlite_db(self, sqlite_db_path):
 
             # Connect and get all table names
             try:
-                with connect(sqlite_db_path) as conn:
-                    cursor = conn.cursor()
-                    cursor.execute('SELECT * from sqlite_master WHERE type = "table"')
-                    tables = cursor.fetchall()
-                    table_names = [table[2] for table in tables]
-
-                    for table_name in table_names:
-                        self._log_sqlite_table(table_name, cursor)
+                self._raw_log_sqlite_db(sqlite_db_path)
 
             except Exception as connection_e:
                 if isinstance(connection_e, OperationalError) and -1 != connection_e.message.find('locked'):
-                    Logger.log_error('!!LOCKED DB!! DID YOU FORGET TO CLOSE CHROME?')
+                    shutil.copyfile(sqlite_db_path, "{0}.tmp".format(sqlite_db_path))
+                    self._raw_log_sqlite_db("{0}.tmp".format(sqlite_db_path))
+                    os.remove("{0}.tmp".format(sqlite_db_path))
 
-                Logger.log_exception(connection_e, message='failed _log_sqlite_db')
+                Logger.log_warning('{0} was locked. Copied to {0}.tmp & analyzed.'.format(sqlite_db_path))
```

Comment:
This is logging on every exception, even when DB was not locked. Move the new warning into the if above and keep the old log_exception in an else?